### PR TITLE
@ #580 | use LANG over prefered language

### PR DESCRIPTION
**Pull request by datphan**
connect #580

### CODE REVIEW

**Comment by hoatle** [comment]
need @hoavt to confirm if the change works.

File: lib/teracy-dev/location/git_synch.rb [hunk 1]
```diff
@@ -10,7 +10,17 @@ class GitWarn < StandardError
...
     class GitSynch < Synch
+      def initialize
+        super
 
+        current_git_version = `git --version`.gsub("git version", "").strip
+
+        required_git_version = '>= 2.20'
+
+        if !TeracyDev::Util.require_version_valid? current_git_version, required_git_version
+          @logger.warn("Your current git version (#{current_git_version}) is not meet the required version (#{required_git_version}), please upgrade it to run properly.")
```
Comment (hoatle) [inline lib/teracy-dev/location/git_synch.rb]: does not meet

**Comment by datphan** [comment]
its work with `ENV['LANG'] = 'en_US'`, not sure it we have to reset it after sync, but I think its fine for now

**Review by hoavt — APPROVED** [review]

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**